src/components/VSourcesTable.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <table
+    :aria-label="$t('sources.aria.table').toString()"
+    role="region"
+    class="table table-fixed w-full mt-4 mb-10 not-prose text-base"
+  >
+    <thead>
+      <tr>
+        <th
+          tabindex="0"
+          @click="sortTable('display_name')"
+          @keypress.enter="sortTable('display_name')"
+        >
+          <span class="w-full flex flex-row items-center justify-between">
+            {{ $t('sources.providers.source') }}
+            <TableSortIcon :active="sorting.field === 'display_name'" />
+          </span>
+        </th>
+        <th
+          tabindex="0"
+          @click="sortTable('source_url')"
+          @keypress.enter="sortTable('source_url')"
+        >
+          <span class="w-full flex flex-row items-center justify-between">
+            {{ $t('sources.providers.domain') }}
+            <TableSortIcon :active="sorting.field === 'source_url'" />
+          </span>
+        </th>
+        <th
+          tabindex="0"
+          @click="sortTable('media_count')"
+          @keypress.enter="sortTable('media_count')"
+        >
+          <span class="w-full flex flex-row items-center justify-between">
+            {{ $t('sources.providers.item') }}
+            <TableSortIcon :active="sorting.field === 'media_count'" />
+          </span>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(provider, index) in sortedProviders" :key="index">
+        <td>
+          {{ provider.display_name }}
+        </td>
+        <td class="font-semibold truncate">
+          <VLink :href="provider.source_url">
+            {{ cleanSourceUrlForPresentation(provider.source_url) }}
+          </VLink>
+        </td>
+        <td class="text-right">
+          {{ getLocaleFormattedNumber(provider.media_count || 0) }}
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</template>
+
+<script lang="ts">
+import sortBy from 'lodash.sortby'
```

Comment:
TIL `lodash.sortby` is over 2300 LOC :scream: Good thing we have #1108 :grin:

Reply:
🤯